javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -778,7 +775,6 @@ public void slang_persistent(Blackhole bh) {
             for (int i = 1; !values.isEmpty(); i++) {
                 values = values.slice(1, values.size());
                 values = values.slice(0, values.size() - 1);
-                assert sizeOf(values.trie) < (branchingFactor() * sizeOf(copyRange(ELEMENTS, i, javaMutable.size() - i))); /* detect memory leak */
```

Comment:
slowed down tests too much (in debug mode only, didn't affect the actual benchmarks)